manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: CheatBreaker.CheatBreaker
+PackageVersion: 2.4.2
+PackageLocale: en-US
+Publisher: Not affiliated with CheatBreaker, LLC

```

```suggestion
Publisher: Cheatbreaker
```
This should match what the application writes to control panel